components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -926,6 +931,9 @@ document.observe("xwiki:dom:loaded", function() {
   font-size: .87em;
   float: right;
 }
+#access-rights-summary .owner {
+  color: inherit;
```

Comment:
Wouldn't work, it effects the modify permissions button ```.access-rights-info a``` works and is a bit more generic I suppose